cmd/krel/cmd/release_notes.go

```diff
@@ -85,6 +96,38 @@ func init() {
 		"version tag for the notes",
 	)
 
+	releaseNotesCmd.PersistentFlags().StringVarP(
+		&releaseNotesOpts.draftOrg,
+		"draft-org",
+		"",
+		"",
+		"a Github organization or user where the Release Notes PR will be created",
```

Comment:
We should mention that this is the org of the fork. (what a rhyme)

Reply:
Done, the new message reads:
```
a Github organization ownwer of the fork of k/sig-release where the Release Notes Draft PR will be created
```